app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -77,29 +84,33 @@ public function handle(Rogue $rogue)
 
                 $post = $rogue->createPost($postData);
                 info('post created in rogue for ' . $this->record['Email address']);
+            // Else if post exists, update post status if required.
             } else {
+                $postId = $fetchPostRes['data'][0]['id'];
+                info($postType.' post '.$postId.' found for user ' . $userId.' and campaign '.$campaignId);
                 $newStatus = $this->translateStatus($this->record['Status'], $this->record['Finish with State']);
-                $statusShouldChange = $this->updateStatus($post['data'][0]['status'], $newStatus);
+                $statusShouldChange = $this->updateStatus($fetchPostRes['data'][0]['status'], $newStatus);
 
                 if ($statusShouldChange) {
-                    $rogue->updatePost($post['data'][0]['id'], ['status' => $statusShouldChange]);
+                    $rogue->updatePost($postId, ['status' => $statusShouldChange]);
                 }
             }
         }
     }
 
     /*
-     * Determines if a record should be process to be stored or if it is not valid.
+     * Returns whether a record is test data that should not create/update users and/or posts.
+     * TODO: Move this into helpers and DRY with any other Jobs that are still relevant.
```

Comment:
Outside of the scope of this PR, but should we delete the TurboVote job if it's no longer used? @ngjo

Reply:
Let's hold off for a little while longer -- we get some straggler registrations so while I've used it sparingly, there has been use for it. I created an icebox ticket tho for it https://www.pivotaltracker.com/story/show/164114650